eRecepta-projektDyplomowy/eRecepta-projektDyplomowy.csproj

```diff
@@ -8,6 +8,7 @@
     <SpaRoot>ClientApp\</SpaRoot>
     <DefaultItemExcludes>$(DefaultItemExcludes);$(SpaRoot)node_modules\**</DefaultItemExcludes>
     <RootNamespace>eRecepta_projektDyplomowy</RootNamespace>
+    <UserSecretsId>572a7511-7dbb-48ab-9384-ecdd2a092c46</UserSecretsId>
```

Comment:
To się zrobiło przez ten secret init, pewnie jak ktoś sklonuje i zrobi u siebie komendę, to pojawi się inne id. Na razie zmergujemy, a ja przy następnym commicie to usunę i dodam do gitignore (tam definiujemy pliki, które mają być ignorowane przez git).